documentation/src/docs/security/securing-standalone-mode-cluster.md

```diff
@@ -0,0 +1,158 @@
+
+Similarly, the configuration parameter `singlenode.enableAuth` determines whether `auth` is enabled. It is disabled by default as well.
+
+The following steps explain how to enable and configure SSL/TLS and/ `auth`:
+
+1. Locate the `standalone-config.properties` file.
+
+   If you are running standalone mode cluster from source, you can find it under the `/path/to/pravega/config` directory.
+
+   If you are running it from the distribution instead, you can find it under the `/path/to/pravega-<version>/conf` directory.
+
+2. If enabling SSL/TLS, optionally prepare the certificates and keys for SSL/TLS.
+
+   In the same directory as the `standalone-config.properties` file, you'll find default/provided PKI material. You may use these material for SSL/TLS.
+
+   If you'd like to create your own material instead, you may follow similar steps as [those](https://github.com/pravega/pravega/wiki/Creating-TLS-material-for-Pravega-security) used for creating the supplied material.
+
+2. If enabling SSL/TLS, configure SSL/TLS parameters.
+
+   If you are using custom material, replace the paths of the files accordingly.
+
+   ```java
+   singlenode.enableTls=true
+   singlenode.keyFile=../config/server-key.key
+   singlenode.certFile=../config/server-cert.crt
+   singlenode.keyStoreJKS=../config/server.keystore.jks
+   singlenode.keyStoreJKSPasswordFile=../config/server.keystore.jks.passwd
+   singlenode.trustStoreJKS=../config/client.truststore.jks
+   ```
+   For descriptions of these properties, see [Pravega Security Configurations](security-configurations.md).  
+
+3. If enabling `auth`, configure `auth` parameters.
+
+   ```java
+   singlenode.enableAuth=true
+   singlenode.userName=admin
+   singlenode.passwd=1111_aaaa
+   singlenode.passwdFile=../config/passwd
+   ```
+   For descriptions of these properties, see [Pravega Security Configurations](security-configurations.md).
+
+   Note:
+   * The default Password Auth Handler supports `Basic` authentication for client-server communications. You may deploy additional Auth Handler plugins by placing them in the classpath.
+
+   * The default handler's configuration file shown above - `passwd` file - contains a single user with credentials containing username `admin` and password `1111_aaaa`.
+
+   * To create additional users or change the password for the default `admin` user:
+
+     a) Create a file containing 1 or more entries of the form  `<user>:<password>:<acl>;`.
+
+        For example: `jdoe:strong-password:*,READ_UPDATE;`
+
+     b) Use the PasswordCreatorTool to create a new file with the passwords encrypted. Configure this new file instead of the supplied `passwd` file.
+
+4. If enabling SSL/TLS, ensure that the server's certificate is trusted.
+
+   Note: This step is only needed the first time you run SSL/TLS-enabled standalone mode with a certificate.If you have already imported the certificate earlier, you don't need to do it again the next time you launch the server.
```

Comment:
Add a space between sentences (`certificate. If`)

Reply:
Done. You sure have a keen eye. :)